examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -94,9 +93,14 @@ describe('Crowdfunding Tests', function () {
     const creationFlags = Object.assign({}, flags);
 
     // create application
-    applicationId = await runtime.addApp({ ...creationFlags, appArgs: creationArgs }, {}, program);
+    applicationId = runtime.addApp({ ...creationFlags, appArgs: creationArgs }, {}, program);
     const creatorPk = addressToPk(creator.address);
 
+    // setup escrow account
+    const escrowProg = getProgram('crowdFundEscrow.py', { APP_ID: applicationId });
+    const lsig = runtime.getLogicSig(escrowProg, []);
```

Comment:
Can we move these 2 lines during setUp of tests? (Maybe declare globally, or in `beforeEach(..)` hook if logic signature will be different in each test). I think this way we don't need to pass address in `syncAccounts()` also

Reply:
I am not sure that's the right way, test should be independent of each other, what if there are two different logic signatures ? We xan modify test to use them before each is more general, can only take same program name.

Reply:
updated @ratik21 please check the updated one.